test/javascript/components/interchange.js

```diff
@@ -185,10 +183,14 @@ describe('Interchange', function() {
         $(window).trigger('resize');
       }, 10);
 
+      setTimeout(function() {
+        $(window).trigger('resize');
+      }, 20);
+
       setTimeout(function() { // Wait for third trigger...
         sinon.assert.calledOnce(spy);
         done();
-      }, 50);
+      }, 150);
```

Comment:
Why `150` instead of `50` ? There should be no race condition there.

Reply:
Well, it seems the tests run slower on Travis and we might set this lower later again but so far this + the other changes fixed it in my other PR which includes all changes.

Reply:
I don't like having magic numbers like this one. There is no reasons for `trigger('resize')` to not trigger `_reflow` immediately.

Reply:
Reset it to `50`.

Reply:
My point is that, according what I understand from the code, `1`, `2`, `3`, etc should be enough as delays, or even `setTimeout` without delays. If there is race conditions with that, well there's a bug in our code and we should not try to hide it in our tests.